src/content/structured/components/toggle-buttons/code.mdx

```diff
@@ -0,0 +1,470 @@
+    <SlottedSVG
+      slot="icon"
+      xmlns="http://www.w3.org/2000/svg"
+      height="24px"
+      viewBox="0 0 24 24"
+      width="24px"
+      fill="#000000"
+    >
+      <path d="M2,21V19H20V21H2M20,8V5H18V8H20M20,3A2,2 0 0,1 22,5V8A2,2 0 0,1 20,10H18V13A4,4 0 0,1 14,17H8A4,4 0 0,1 4,13V3H20M16,5H6V13A2,2 0 0,0 8,15H14A2,2 0 0,0 16,13V5Z" />
+    </SlottedSVG>
+  </IcToggleButton>
+</ComponentPreview>
+
+### Sizing
+
+export const snippetsSizing = [
+  {
+    language: "Web component",
+    snippet: `<ic-toggle-button label="Switch to oat milk" size="small"></ic-toggle-button>
+<ic-toggle-button label="Switch to oat milk"></ic-toggle-button>
+<ic-toggle-button label="Switch to oat milk" size="large"></ic-toggle-button>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcToggleButton label="Switch to oat milk" size="small" />
+<IcToggleButton label="Switch to oat milk" size="default" />
+<IcToggleButton label="Switch to oat milk" size="large" />`,
+  },
+];
+
+<ComponentPreview style={{ gap: "8px" }} snippets={snippetsSizing}>
+  <IcToggleButton label="Switch to oat milk" size="small" />
+  <IcToggleButton label="Switch to oat milk" size="default" />
+  <IcToggleButton label="Switch to oat milk" size="large" />
+</ComponentPreview>
+
+### With badge
+
+export const snippetsWithBadge = [
+  {
+    language: "Web component",
+    snippet: `<ic-toggle-button label="Switch to oat milk">
+    <ic-badge text-label="1" slot="badge" variant="success"></ic-badge>
+</ic-toggle-button>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcToggleButton label="Switch to oat milk">
+    <IcBadge textLabel="1" slot="badge" variant="success" />
+</IcToggleButton>`,
+  },
+];
+
+<ComponentPreview snippets={snippetsWithBadge}>
+  <IcToggleButton label="Switch to oat milk">
+    <IcBadge textLabel="1" slot="badge" variant="success" />
+  </IcToggleButton>
+</ComponentPreview>
+
+### Appearance dark
+
+export const snippetsDark = [
+  {
+    language: "Web component",
+    snippet: `<ic-toggle-button label="Switch to oat milk" appearance="dark"></ic-toggle-button>
+<ic-toggle-button label="Switch to oat milk" appearance="dark" loading="true"></ic-toggle-button>
+<ic-toggle-button label="Switch to oat milk" appearance="dark" loading="true" toggle-checked="true"></ic-toggle-button>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcToggleButton label="Switch to oat milk" appearance="dark" />
+<IcToggleButton label="Switch to oat milk" appearance="dark" loading />
+<IcToggleButton label="Switch to oat milk" appearance="dark" loading toggleChecked />`,
+  },
+];
+
+<ComponentPreview style={{ gap: "8px" }} snippets={snippetsDark}>
+  <IcToggleButton label="Switch to oat milk" appearance="dark" />
+  <IcToggleButton label="Switch to oat milk" appearance="dark" loading />
+  <IcToggleButton
+    label="Switch to oat milk"
+    appearance="dark"
+    loading
+    toggleChecked
+  />
+</ComponentPreview>
+
+### Appearance light
```

Comment:
```suggestion
### Light
```
I had a look at other component pages that use the `appearance` prop. Typically, when we split the appearance code previews up, we title them light and dark (see Tabs and Skeleton)